app/public/javascripts/calendar.js

```diff
@@ -46,20 +58,20 @@ var Calendar = function() {
 		}
 	}
 
-	function dayNums() {
-			var numRows = 5;
-			var numCols = 7;
-		    for (var r=0, id=1; r<numRows; ++r) {
-	           	var tr = document.createElement('tr');
-	            $(".curr").append(tr);
-	           	for (var c=0; c<numCols; ++c, ++id) {
-	                var td = document.createElement('td');
-	                td.setAttribute('id', id);
-	                tr.appendChild(td);
-	            }
-	        }
-
-	}
+	// function dayNums() {
```

Comment:
In general, I would recommend deleting code over commenting it out.  We can always go an get it back through git history if necessary.